package.js

```diff
@@ -1,6 +1,6 @@
 Package.describe({
   "summary": "Server Side Rendering for Meteor with Blaze",
-  "version": "2.1.1",
+  "version": "2.2.0",
```

Comment:
Oh! I missed this.
No. We'll do it. But I do it in a different commit. 
I always do the releases and increase numbers.
Anyway, I'll take this.